requirements.txt

```
@@ -0,0 +1,8 @@
+akari-client[depthai]
+akari-proto
+ndjson
+tqdm
+playsound
+opencv-python >= 4.5.1.48
+open3d
+depthai >= 2.10
```

```suggestion
depthai==2.30.0
```
9/8にdepthai3.0.0がリリースされ、最新のdepthaiで既存の実装は動作しなくなりました…。
従ってdepthaiのバージョンを2に固定する必要があります。